ops/ansible/playbooks-ccs/vars/test/group_vars/all/env_specific.yml

```diff
@@ -20,6 +20,9 @@ data_pipeline_idempotency_required: false
 # Whether to enable BFD API V2
 data_server_v2_enabled: true
 
+# Whether to enable BFD PreAdj Resources
+pre_adj_resources_enabled: false
```

Comment:
@KyRobbins depending on the order in which the PRs get merged (this, search and transformers), we can set this to `true`.